### DISCUSSION TITLE
Make the "Repeat" and "Random" settings related to playlists

### DISCUSSION THREAD

**Discussion by Patakreyp** [strawberrymusicplayer/strawberry#1232] [Ideas]
The idea would be that you could save the repeat and random setting with each playlist instead of each being a global setting.

For example, you could make a playlist with all Pink floyd albums and make it repeatable in order without having to bother to chnage the Repeat/Random setting, and if you switch to another playlist (e.g. containing all your albums) make it a pseudo-radio playing songs at random.

**Comment by jonaski** [comment]
https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
